### PR TITLE
Allinone name changes in run_checks

### PR DESCRIPTION
**Pull request by eneskuluk**
ek_exclude_spark branch in GenomicsDB fails at release_test phase since run_checks.sh looking for allinone.jar instead of allinone-spark.jar.
I think this should be merged after making sure https://github.com/GenomicsDB/GenomicsDB/pull/281 can be merged since it might affect other branches that doesn't include -spark change and has a tag.

### CODE REVIEW

**Comment by eneskuluk** [comment]
I'm not sure if test_genomicsdbjar.sh needs to change as well since it uses statically version 1.4.3 and pulls from master or I might be missing something.

**Review by nalinigans — CHANGES_REQUESTED** [review]
Just wondering if we can look for either allinone.jar or allinone-spark.jar in this script?

**Comment by nalinigans** [comment]
> I'm not sure if test_genomicsdbjar.sh needs to change as well since it uses statically version 1.4.3 and pulls from master or I might be missing something.

Yes, `test_genomicsdbjar.sh` needs some changing as well to work from the `GenomicsDB` workflows. Let me see what I can do here. Thanks.

**Comment by eneskuluk** [comment]
> Just wondering if we can look for either allinone.jar or allinone-spark.jar in this script?

Yes sure, that makes so much sense. I'll fix it.

**Comment by mlathara** [comment]
> I'm not sure if test_genomicsdbjar.sh needs to change as well since it uses statically version 1.4.3 and pulls from master or I might be missing something.

So, it looks like I have an old branch https://github.com/GenomicsDB/TestGenomicsDBJar/tree/ml_genomicsdb_release which needs to be pulled in. The current release_test workflow checks out that branch...not the master branch of TestGenomicsDBJar.

**Comment by eneskuluk** [comment]
Currently it should be checking for both @nalinigans

File: run_checks.sh [hunk 1]
```diff
@@ -1,15 +1,22 @@
-#!/bin/bash 
-if [[ ! -f genomicsdb-${GENOMICSDB_VERSION}-allinone.jar ]]; then
-  echo "Could not find genomicsdb-${GENOMICSDB_VERSION}-allinone.jar"
-  exit 1
+#!/bin/bash
+file_size_kb=0
+if [[ ! -f genomicsdb-${GENOMICSDB_VERSION}-allinone-spark.jar ]]; then
```
Comment (nalinigans) [inline run_checks.sh]: Can you change the block from Line 3 onwards to something like this?
```bash
if [[ ! -f genomicsdb-${GENOMICSDB_VERSION}-allinone-spark.jar && ! -f genomicsdb-${GENOMICSDB_VERSION}-allinone.jar ]]; then
  echo "Could not find genomicsdb-${GENOMICSDB_VERSION}-allinone-spark.jar or genomicsdb-${GENOMICSDB_VERSION}-allinone-spark.jar"
 	exit 1
fi

if [[ -f genomicsdb-${GENOMICSDB_VERSION}-allinone.jar ]]; then
	file_size_kb=`du -k genomicsdb-${GENOMICSDB_VERSION}-allinone.jar | cut -f1`
else
	file_size_kb=`du -k genomicsdb-${GENOMICSDB_VERSION}-allinone-spark.jar | cut -f1`
fi
```

**Review by nalinigans — CHANGES_REQUESTED** [review]

File: run_checks.sh (hunk 1 above)
Comment (eneskuluk) [reply run_checks.sh]: Sure, that looks good.

**Review by nalinigans — APPROVED** [review]

**Pull request merged**